src/subscription.ts

```diff
@@ -53,7 +53,7 @@ export class Subscription {
 
   // Public, but for internal use
   public close() {
-    return new Promise((resolve, reject) => {
+    return new Promise<void>((resolve, reject) => {
```

Comment:
Doesn't hurt older versions of TypeScript, but is required for more recent versions when `resolve` is called with 0 arguments so went ahead and changed it.